src/orchestrator/orchestrator.rs

```diff
@@ -415,7 +425,7 @@ impl<'a> Orchestrator<'a> {
             .find(|j| j.3.borrow().is_none())
             .ok_or_else(|| anyhow!("Failed to find root task"))?;
         let root_job_id = root_job.1.jobdef.job.uuid();
-        trace!("Root job id = {}", root_job_id);
+        trace!(%root_job_id, "Root job id found");
```

Comment:
This is _structured_ data attached to your trace. You can, when inspecting the trace output of a program (after the fact or while the program is running) filter for this data. For example you can filter for `root_job_uuid=<some uuid>` to only see traces related to that uuid.

Reply:
Ok, yes, I was just wondering if it would make sense to call the field `job_uuid` instead of `root_job_id` since this is the only event where we use `root_job_id` but it certainly doesn't hurt to use structured fields here.

Anyway, this should be fine for now - we can always change it later depending on our filtering needs :)